apps/web/src/app/auth/settings/two-factor/two-factor-setup-duo.component.html

```diff
@@ -13,7 +13,7 @@
         <strong>{{ "twoFactorDuoApiHostname" | i18n }}:</strong> {{ host }}
       </ng-container>
       <ng-container *ngIf="!enabled">
-        <img class="tw-float-right tw-ml-3 mfaType2" alt="Duo logo" />
+        <img class="tw-float-right tw-ml-3" alt="Duo logo" />
```

Comment:
I think we still need `mfaType2` here.

```suggestion
        <img class="tw-float-right tw-ml-3 mfaType2" alt="Duo logo" />
```